src/content/development/shipyard/settings/_index.en.md

```diff
@@ -33,6 +33,7 @@ The possible settings are:
     and **`ovn`**.
   * **`nodes`**: A space separated list of nodes to deploy, supported types are **`control-plane`** and **`worker`**.
   * **`submariner`**: If Submariner should be deployed, set to **`true`**. Otherwise, leave unset (or set to **`false`** explicitly).
+  * **`gateways`**: Number of gateway nodes that needs to be deployed globally or on per cluster level.
```

Comment:
This is in the “Global and/or cluster specific” section, so “globally or on per cluster level” is specified by the section:

```suggestion
  * **`gateways`**: Number of gateway nodes to deploy.
```

Reply:
Done